ethstore/README.md

```diff
@@ -20,6 +20,7 @@ Usage:
     ethstore change-pwd <address> <old-pwd> <new-pwd> [--dir DIR]
     ethstore list [--dir DIR]
     ethstore import [--src DIR] [--dir DIR]
+    ethstore import-wallet <path> <password> [--dir DIR]
```

Comment:
bit horrible putting a password in the CLI invocation. better to have an interactive password prompt